doc/interactivity/CMakeLists.txt

```diff
@@ -12,11 +12,11 @@ add_library(${INTERACTIVITY_LIB_NAME}
   # src/imarker.cpp
   # src/pose_string.cpp
 # )
-# target_link_libraries(interactivity_tutorial
-  # ${catkin_LIBRARIES}
-  # ${Boost_LIBRARIES}
-  # interactive_markers
-# )
+target_link_libraries(${INTERACTIVITY_LIB_NAME}
+  ${catkin_LIBRARIES}
+  ${Boost_LIBRARIES}
+  interactive_markers
+)
```

Comment:
there is already a `target_link_libraries` directive for this below.
Please move the link_libraries directive right behind the library definition above if you really need to change things here.

`interactive_markers` is an external package, so catkin should ensure that it's libraries are listed in `${catkin_LIBRARIES}`.
Please check.
Maybe there is a `<build_depend>interactive_markers</build_depend>` missing in some package or dependency and catkin gets the order wrong?